### ISSUE TITLE
Alternative to Alt+Del in Jaws - position of cursor in MS Word

### ISSUE THREAD

**Issue by jaxcze** [nvaccess/nvda#6352]
In Jaws screenreader is shortcut Alt+Del. This one says position of cursor from left edge of paper or printing mirror in MS Word.
This is missing in NVDA.

**Comment by jcsteh** [comment]
Duplicate of #3707.
